### PR TITLE
feature/L7 favorite data type

### PR DESCRIPTION
**Pull request by AlexRecross**
- page with my favorit data type of **JS**

### CODE REVIEW

File: favorit_js_data_type/index.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>My fovorite data type in JS</title>
+</head>
+<body bgcolor="#f6f8fa">
+	<h1  bordertype="solid">Arrays</h1>
+	<p>My favorite data type is an array! Because it can contain any types of data of any quantity, including arrays)</p>
```
Comment (sajera) [inline favorit_js_data_type/index.html]: Seems good, but the object also may contain any type of JS data types. Why the Array preferred for you ?

**Review by sajera — CHANGES_REQUESTED** [review]
Please take a look on review issues

File: favorit_js_data_type/index.html (hunk 1 above)
Comment (AlexRecross) [reply favorit_js_data_type/index.html]: In contrast ordinary objects, the contents of an array are ordered

**Review by sajera — APPROVED** [review]

File: favorit_js_data_type/index.html (hunk 1 above)
Comment (AlexRecross) [reply favorit_js_data_type/index.html]: OK. Take a look on updates

**Pull request merged**